docs/Developers/Meta-review.rst

```diff
@@ -0,0 +1,140 @@
+Meta-Reviewing
+==============
+
+This document is a guide to coala's meta-review process.
+
+What is Meta-review?
+--------------------
+
+People respond to review comments on pull requests by giving feedback. Emoji
+make it much easier to give and receive feedback without a long comment thread.
+Those emoji are called
+`reactions <https://developer.github.com/v4/enum/reactioncontent/>`__ on GitHub
+or `award emoji <https://docs.gitlab.com/ee/user/award_emojis.html>`__ on GitLab.
+They are used by people to publicly express their feelings about review
+comments, which provides feedback to comment authors and all other members.
+
+There are 6 types of reactions on GitHub:
+
+============  =============  =======  =====
+ Name         Markdown       Unicode  Emoji
+============  =============  =======  =====
+ THUMBS_UP    ``:+1:``       U+1F44D  👍
+ THUMBS_DOWN  ``:-1:``       U+1F44E  👎
+ LAUGH        ``:smile:``    U+1F604  😄
+ HOORAY       ``:tada:``     U+1F389  🎉
+ CONFUSED     ``:confused``  U+1F615  😕
+ HEART        ``:heart:``    U+FE0F   ❤️
+============  =============  =======  =====
+
+These reactions are reviews to reviews, thus we call them meta-reviews. To
+encourage people to do meta-reviews, we build a meta-review system, which
+collects reactions, analyze them, and use that information to score people.
+After scoring, a ranking list is shown on coala community website.
+
+At the moment, only THUMBS_UP and THUMBS_DOWN are collected and analyzed,
+as they are most commonly used and express strong and clear feelings:
+THUMBS_UP means the review comment is well-written and helpful, while
+THUMBS_DOWN means the review comment is misleading, or even worse, violates
+`coala Community Code of Conduct <https://github.com/coala/cEPs/blob/master/cEP-0006.md>`__.
+
+Meta-review Process
+-------------------
+
+The meta-review process for coala is as follows:
+
+1. Reviewers create their reviews.
+
+2. Anyone especially author of the pull request can do meta-review (THUMBS_UP,
+   THUMBS_DOWN) via reactions on GitHub.
+
+3. Meta-review information will be collected and analyzed automatically:
+
+   * Meta-reviewers will be given bonus points to encourage such behaviour
+
+   * Meta-reviewees will be given positive points (on receiving THUMBS_UP) or
+     negative points (on receiving THUMBS_DOWN). Those points are weighted
+     according to score of meta-reviewers. Those who have higher score in
+     the meta-review system will have more impacts on others, so please do
+     meta-reviews carefully.
+
+.. note::
+
+    Don't edit or even delete your review after it has been meta-reviewed.
```

Comment:
I think you can just say 'Don't edit or delete ' with "edit" and "delete" in italics or bold. No need for the "even"